qiita_db/analysis.py

```diff
@@ -1011,6 +1011,10 @@ def _build_mapping_file(self, samples, rename_dup_samples=False):
                     samps = set(samps) - all_ids
                     all_ids.update(samps)
 
+                # appending the study title
+                qm['qiita_study_title'] = qdb.artifact.Artifact(
```

Comment:
Historically (I think) the name of the column has varied from `title` to `study_title`, etc, do we want to add yet another column name? I think the answer maybe is yes, because some studies have a legacy `title` column that is inaccurate right?